Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx

```diff
@@ -56,12 +57,18 @@ export default class AssignDatapoints extends React.Component {
             <div className="device-details">
               <p>{deviceData.name}</p>
               <p>
-                {datapointsData.length} {strings.datapointAssigned}
+                <span>
+                  {datapointsData.length} {strings.datapointAssigned}
+                </span>
+                <span className="divider">.</span>
+                <a href="#" onClick={() => this.props.changeTab('EDIT_DATAPOINTS', deviceData.id)}>
+                  Edit
```

Comment:
It feels a little strange that the Edit button is active even when there are no datapoints assigned. This should probably only be activated when there are actually datapoints that have been assigned to a device.

Reply:
Also should be a translated string.